src/portals/hope/features/claims/claim-details/ClaimSubclaims/components/AddSubclaim.tsx

```diff
@@ -95,18 +96,18 @@ gql`
   }
 `
 
-export const AddSubclaim: React.FC<{ claimId: string; memberId: string }> = ({
-  claimId,
-  memberId,
-}) => {
+export const AddSubclaim: React.FC<{
+  claimId: string
+  memberId: string
+}> = ({ claimId, memberId }) => {
   const [addSubclaim] = useAddSubclaimMutation()
-  const { data: claimTypesData } = useGetClaimTypesQuery()
   const { data: contractsData } = useMemberContractsQuery({
     variables: { memberId },
   })
+  const { data: typesData } = useGetClaimTypesQuery()
 
+  const claimTypes = typesData?.claimTypes ?? []
```

Comment:
We don't need this anymore
```suggestion
  const availableClaimTypes = selectedContract?.availableClaimTypes ?? []
```